libtransmission/global-ip-cache.h

```diff
@@ -94,19 +119,11 @@ class tr_global_ip_cache
     [[nodiscard]] bool set_is_updating(tr_address_type type) noexcept;
     void unset_is_updating(tr_address_type type) noexcept;
 
-    void update_global_addr(tr_address_type type) noexcept;
-    void update_source_addr(tr_address_type type) noexcept;
-
-    // Only use as a callback for web_->fetch()
-    void on_response_ip_query(tr_address_type type, tr_web::FetchResponse const& response) noexcept;
-
-    tr_web& web_;
-
-    array_ip_t<std::optional<tr_address>> settings_bind_addr_;
+    Mediator& mediator_;
```

Comment:
Can't do a full review tonight, but this stuck out and I thought I'd best write it down before I forget

`mediator_` should be private

Reply:
It is private.